orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -187,7 +191,14 @@ class OperationsController {
   private Map<String, String> startPipeline(Map config) {
     def json = objectMapper.writeValueAsString(config)
     log.info('requested pipeline: {}', json)
-    def pipeline = pipelineStarter.start(json)
+
+    def pipeline
+    if (config.executionEngine == "v2") {
```

Comment:
Just needed some way of identifying a pipeline that should run against the "new" engine (engine isn't quite the right wording for where we're at but what the hell)